decidim-templates/config/locales/en.yml

```diff
@@ -46,7 +46,6 @@ en:
         block_user_templates:
           form:
             save: Save
-            template_title: Template information
```

Comment:
I agree removing this from the template as it isn't adding much information. 

The thing is that we're removing it only from one of the views, (for `block_user_templates`). Can you remove this also from `questionnaire_templates` 🙏🏽 ?

Reply:
Fixed in 87491df4ac